app/assets/stylesheets/modules/record-metadata-section.scss

```diff
@@ -2,15 +2,38 @@
   .section {
     padding-bottom: 0;
   }
+
+  h3 {
+    color: $smoke;
+  }
 }
 
-.record-metadata, .upper-record-metadata {
+.record-metadata,
+.upper-record-metadata,
+.purl-embed-viewer {
   @include show-page-row-layout;
+
+  .section-heading {
+    h3 {
+      color: $smoke;
```

Comment:
We'll be swapping this color out for `$stone` once we merge in changes from `master`.

Reply:
✅ resolved by #2195